openproblems/tasks/_batch_integration/batch_integration_graph/methods/scanvi.py

```diff
@@ -0,0 +1,21 @@
+# from ....tools.normalize import log_cpm
+from .....tools.decorators import method
+
+# from ....tools.utils import check_version
+from scIB.integration import runScanvi
+from scIB.preprocessing import reduce_data
+
+
+@method(
+    method_name="scanvi",
+    paper_name="",
+    paper_url="",
+    paper_year=0,
+    code_url="",
+    # code_version=check_version("numpy"),
+    # image="openproblems-template-image" # only if required
+)
+def scvi_emb(adata):
+    runScanvi(adata, "batch", "labels")
```

Comment:
need to have an extra check here that count data exist (or mention this in the description as i think it's already in scIB).